### DISCUSSION TITLE
TP4 - Ejercicio 6 - Intercambio de caracteres - consulta

### DISCUSSION THREAD

**Discussion by Nikzza** [INGCOM-UNRN-P1/cursada-2023#135] [Preguntas y respuestas] [answered]
Tengo una duda, en este ejercicio la funcion que se elige como sugerencia no entra ningun tipo de puntero a la funcion. Eso esta correcto?

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141664052/dc15324c-bf01-46c9-a849-108909f0ffec)

**Comment by martinvilu** [comment]
> ningun tipo de puntero a la funcion

¿Ninguno? Yo creo que si

**Reply by martinvilu** [reply, marked as answer]
Siendo que, por un lado, todo tiene una dirección en memoria accesible por el operador `&`, y que `cadena` es un bloque de memoria.

No hace falta que sea que `cadena` _explicitamente_ un puntero, que de todas formas, ya funciona como uno.

**Reply by Nikzza** [reply]
ta perfecto, ahora entendi

**Comment by martinvilu** [comment]
![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/6e4392ff-a585-4105-b9a7-fdbe402378b8)
Le hice click _instintivamente_ 🤣
